third-party/dnd-wiki/classes/class-weapon-master.xml

```diff
@@ -0,0 +1,454 @@
+			<p>You gain a +2 bonus to attack rolls you make with ranged weapons.</p>
+			<h5>DUELING</h5>
+			<p>When you are wielding a melee weapon in one hand and no other weapons, you gain a +2 bonus to damage rolls with that weapon.</p>
+			<h5>GREAT WEAPON FIGHTING</h5>
+			<p>When you roll a 1 or 2 on a damage die for an attack you make with a melee weapon that you are wielding with two hands, you can reroll the die and must use the new roll, even if the new roll is a 1 or a 2. The weapon must have the two-handed or versatile property for you to gain this benefit.</p>
+			<h5>TWO-WEAPON FIGHTING</h5>
+			<p>When you engage in two-weapon fighting, you can add your ability modifier to the damage of the second attack.</p>
+		</description>
+		<sheet />
+		<rules>
+			<select type="Class Feature" name="Fighting Style" supports="Fighting Style" level="1"/>
+		</rules>
+	</element>
+	<element name="Martial Archetype" type="Class Feature" source="D&amp;D Wiki" id="ID_WIKI_CLASS_FEATURE_WEAPON_MASTER_MARTIAL_ARCHETYPE">
+		<description>
+			<p>At 3rd level, you choose the path that you walk as a Weapon master. Choose Combat Mastery, Tactic Mastery, or Evasive Mastery all detailed at the end of the class description. The pathway you choose grant you features at 3rd level and again at 7th, 10th, 15th, and 18th level.</p>
+		</description>
+		<sheet>
+			<description></description>
+		</sheet>
+		<rules>
+			<select type="Archetype" name="Weapon Master Martial Archetypes" supports="Weapon Master Martial Archetypes" />
+		</rules>
+	</element>
+	<element name="Ability Score Improvement (Weapon Master)" type="Class Feature" source="D&amp;D Wiki" id="ID_WIKI_CLASS_FEATURE_WEAPON_MASTER_ABILITYSCOREIMPROVEMENT">
+		<description>
+			<p>When you reach 4th level, and again at 8th, 12th, 16th, and 19th level, you can increase one ability score of your choice by 2, or you can increase two ability scores of your choice by 1. As normal, you can't increase an ability score above 20 using this feature.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<!-- <grant type="Class Feature" id="ID_INTERNAL_CLASS_FEATURE_ASI_4_WEAPON_MASTER" level="4" requirements="!ID_INTERNAL_OPTION_ALLOW_FEATS" /><grant type="Class Feature" id="ID_INTERNAL_CLASS_FEATURE_ASI_8_WEAPON_MASTER" level="8" requirements="!ID_INTERNAL_OPTION_ALLOW_FEATS" /><grant type="Class Feature" id="ID_INTERNAL_CLASS_FEATURE_ASI_12_WEAPON_MASTER" level="12" requirements="!ID_INTERNAL_OPTION_ALLOW_FEATS" /><grant type="Class Feature" id="ID_INTERNAL_CLASS_FEATURE_ASI_16_WEAPON_MASTER" level="16" requirements="!ID_INTERNAL_OPTION_ALLOW_FEATS" /><grant type="Class Feature" id="ID_INTERNAL_CLASS_FEATURE_ASI_19_WEAPON_MASTER" level="19" requirements="!ID_INTERNAL_OPTION_ALLOW_FEATS" /> -->
+			<select type="Class Feature" name="Improvement Option (Weapon Master 4)" supports="Improvement Option,Weapon Master,4" default="ID_INTERNAL_CLASS_FEATURE_ASI_4_WEAPON_MASTER" level="4" requirements=""/>
```

Comment:
default (and empty requirements) can be removed here, i've removed them from other classes aswell

Reply:
Removed.